services/sleeper/src/sleeper/main.py

```diff
@@ -0,0 +1,110 @@
+import os
+import json
+import random
+import time
+import subprocess
+
+
+def get_from_environ(key, default=None):
+    """Returns a value from the environ if not found None"""
+    return os.environ.get(key, default)
+
+
+def get_random_sleep():
+    """Returns a random amount of sleep between 1 and 9"""
+    return random.randint(1, 9)
+
```

Comment:
same as above. Moreover, this does NOT sleep, therefore there is no "random sleep" but rather a random integer between 1 and 9 :-)